### PR TITLE
CollectedPaths docs and by default collect everything

### LINKED ISSUE #256: CollectedPaths
- [ ] By default collect everywhere (maybe except nette and latte where it is pointless?)
- [ ] Add documentation

### PR DESCRIPTION
**Pull request by MartinMystikJonas**
Resolves #256

### CODE REVIEW

**Comment by lulco** [comment]
I changed my mind here. I want this package to be as fast as possible, so we will analyse only analysedPaths by default. Of course we need documentation how to collect data from additional directories, but that's all. If user gets error about non-existing variable / component he will go throught docs and see how to set another dirs to collected paths, but if he will get no error but analysis will be slow, he will just nervous.

File: README.md [hunk 1]
```diff
@@ -51,6 +51,8 @@ It is recommended to use this extension with [phpstan-nette](https://github.com/
 * [Link checking](docs/configuration.md#link-checking)
 * [Other configuration options](docs/configuration.md#other-configuration-options)
 
+If analysis is slow you should set option `latte.collectedPaths` as desciption in [Analyser configuration](docs/configuration.md#analyser-configuration).
```
Comment (lulco) [inline README.md]: - if you need to collect (not analyse - no errors will be reported) extra files or directories 
- typo description

File: docs/configuration.md [hunk 2]
```diff
@@ -101,6 +101,23 @@ parameters:
 
 ## Analyser configuration
 
+### collectedPaths
+Type: `array`
+
+List of paths outside of analysedPaths that are analysed when latte context (variables, components, ...) is collected for template analysis.
```
Comment (lulco) [inline docs/configuration.md]: They are not analysed in phpstan mean. We just collect data from them

File: docs/configuration.md [hunk 3]
```diff
@@ -101,6 +101,23 @@ parameters:
...
+
+By default all files referenced from analysed code are collected. That can be slow if you have some big vendor directory.
```
Comment (lulco) [inline docs/configuration.md]: Remove this part

**Comment by MartinMystikJonas** [comment]
Ok so keep original implementation just update docs?

**Comment by lulco** [comment]
> Ok so keep original implementation just update docs?

Yes please.

**Review by lulco — APPROVED** [review]

**Comment by MartinMystikJonas** [comment]
Changed to jsut docs update

**Pull request merged**